app/models/query/business_entity_management.rb

```diff
@@ -0,0 +1,13 @@
+module Query
+  class BusinessEntityManagement
+    def initialize(office)
+      @office = office
+    end
+
+    def jurisdictions
+      Jurisdiction.all.map do |jurisdiction|
+        Views::OfficeBusinessEntityState.new(@office, jurisdiction)
```

Comment:
I'm not sure the query object should return view objects, but rather a `ActiveRecord::Relation`? Then the views creation is a responsibility of the controller.